src/java/org/apache/cassandra/config/Config.java

```diff
@@ -869,6 +869,8 @@ public static void setClientMode(boolean clientMode)
     public volatile int minimum_replication_factor_fail_threshold = -1;
     public volatile int maximum_replication_factor_warn_threshold = -1;
     public volatile int maximum_replication_factor_fail_threshold = -1;
+    public volatile boolean zero_ttl_on_twcs_fail = false;
+    public volatile boolean zero_ttl_on_twcs_warn = true;
```

Comment:
I would declare the soft limit (warn) before the hard limit (fail), as in the rest of the guardrail properties.
```suggestion
    public volatile boolean zero_ttl_on_twcs_warn = true;
    public volatile boolean zero_ttl_on_twcs_fail = false;
```